packages/framework/ai-collab/package.json

```diff
@@ -71,12 +71,14 @@
 		"cache-dir": "nyc/.cache",
 		"exclude": [
 			"src/test/**/*.*ts",
-			"dist/test/**/*.*js"
+			"dist/test/**/*.*js",
+			"lib/test/**/*.*js"
 		],
 		"exclude-after-remap": false,
 		"include": [
 			"src/**/*.*ts",
-			"dist/**/*.*js"
+			"dist/**/*.*js",
+			"lib/**/*.*js"
```

Comment:
I think in most coverage config we are just including ESM. I am not familiar with how we are actually using coverage; so, can't say which is better.
I think the `dist` inclusion will be a problem if we skip CJS under `test` script as doing here. I think `dist` needs dropped or `test:mocha` needs to run CJS tests too.

Reply:
Hmm I hadn't considered that, but it doesn't seem to affect numbers? A [recent PR](https://github.com/microsoft/FluidFramework/pull/22746) by @jatgarg made it so we added the lib/ bits where they were missing, but we did not remove any dist/ ones. A quick skim of some packages suggests our standard config includes them both.

So I ran tests with coverage locally for two separate packages (container-loader, counter) both of which skip the CJS tests. If we only have the dist/ bits in the c8 config, the report output says 0% coverage, which I think makes sense, since c8 would only care about the built test files that we didn't run. But whether we have only lib/, or lib/ + dist/, the output is identical. I guess in the end what it cares about is the coverage on the ts files, independently of which build output files actually ran and provided said coverage?

Reply:
Yep - if we roll to .ts, then all should be fine right now...

I think it is more important to have coverage on the .js files that customers will actually be using. Mapping to .ts is convenient for developers but might miss things if there are funny bits with generation - like some complicated typescript generates multiple .js blocks and we only hit a subset. (I think that is more likely to be the case for CJS than ESM.)

I have a little preference to just use ESM as I want to ensure those tests are being run and it is interesting to know what isn't covered by ESM.
Not running CJS tests gives that same effect ... until someone enables them.

I am fine with both or just `lib` given the current state.